packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -23,7 +23,7 @@ import { IResourceMetering } from "src/L1/interfaces/IResourceMetering.sol";
 contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
     /// @notice Enum representing different types of updates.
     /// @custom:value BATCHER              Represents an update to the batcher hash.
-    /// @custom:value GAS_CONFIG           Represents an update to txn fee config on L2.
+    /// @custom:value FEE_SCALARS          Represents an update to l1 data fee scalars.
```

Comment:
Can you also rename it in the `UpdateType`? There may be places in `op-node` that need the name change as well

Reply:
yup will do

Reply:
done